index-de.html

```diff
@@ -0,0 +1,273 @@
+                            <td><label for="GatheringItem">Sammelwerkzeuge</label></td>
+                            <td><input type="checkbox" data-type="Gizmo" id="GizmoItem" class="itemType"></td>
+                            <td><label for="GizmoItem">Gizmos</label></td>
+                            <td><input type="checkbox" data-type="MiniPet" id="MiniPetItem" class="itemType"></td>
+                            <td><label for="MiniPetItem">Miniaturen</label></td>
+                        </tr>
+                        <tr>
+                            <td><input type="checkbox" data-type="Tool" id="ToolItem" class="itemType"></td>
+                            <td><label for="ToolItem">Wiederverwertungskit</label></td>
+                            <td><input type="checkbox" data-type="Trait" id="TraitItem" class="itemType"></td>
+                            <td><label for="TraitItem">Eigenschaftenhandbuch</label></td>
+                            <td><input type="checkbox" data-type="Trinket" id="TrinketItem" class="itemType"></td>
+                            <td><label for="TrinketItem">Schmuck</label></td>
+                        </tr>
+                        <tr>
+                            <td><input type="checkbox" data-type="Trophy" id="TrophyItem" class="itemType"></td>
+                            <td><label for="TrophyItem">Trophäen</label></td>
+                            <td><input type="checkbox" data-type="UpgradeComponent" id="UpgradeComponentItem" class="itemType"></td>
+                            <td><label for="UpgradeComponentItem">Aufwertungen</label></td>
+                            <td><input type="checkbox" data-type="Weapon" id="WeaponItem" class="itemType"></td>
+                            <td><label for="WeaponItem">Waffen</label></td>
+                        </tr>
+                    </tbody>
+                </table>
+                </td>
+            </tr>
+            <tr>
+                <td colspan="3">
+                    <label for="ignoreHidden"><input type="checkbox" id="ignoreHidden"> Verberge den Wert von versteckten Items</label>
+                </td>
+            </tr>
+            </table>
+            
+            <hr>
+            <div id="settingsButtons">
+                <button id="restoreDefaults">Stelle Voreinstellungen wieder her</button>
+                <button class="clearAPIKey">Setze meinen API Schlüssel zurück</button>
+            </div>
+        </div>
+
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.3.min.js"><\/script>')</script>
+        <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
+        <script src="js/vendor/highstock.js"></script>
+        <script src="js/console.js"></script>
+        <script src="js/main.js"></script>
```

Comment:
The german js file should be included here before main.js :
<script src="js/lang/de.js"></script>